### PR TITLE
Create use-fail2ban-on-ubuntu

### PR DESCRIPTION
**Pull request by BadCrumpany**

### CODE REVIEW

**Comment by catlook** [comment]
build me

**Comment by rackernexus** [comment]
Build 2295 was successful. Staging content [Preview!](https://support-revs.staging.deconst.org/build-08937810d/how-to/). valid for 14 days

**Comment by catlook** [comment]
approved

File: content/cloud-servers/use-fail2ban-on-ubuntu.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+### Prerequisites
+
+Before you install Fail2Ban, it is good practice always to ensure your server is up to date.  Execute the following
+command to update the current versions of packages installed on your server:
+
+      root@server-01:~# sudo apt-get upgrade
+
+**Note:**  The preceding command provides a summary of packages to be upgraded and prompts you with the required
+of additional space for the installation.  Type "**y**" to continue with the upgrade.
```
Comment (chri2547) [inline content/cloud-servers/use-fail2ban-on-ubuntu.md:29]: strike 'of'

File: content/cloud-servers/use-fail2ban-on-ubuntu.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+    # A host is banned if it has generated "maxretry" during the last "findtime"
+    # seconds.
+    findtime  = 10m
+
+    # "maxretry" is the number of failures before a host get banned.
+
+    maxretry = 3
+     
+The following example shows three unsuccessful login attempts.  You can see the IP address is now banned as proof of concept.
```
Comment (chri2547) [inline content/cloud-servers/use-fail2ban-on-ubuntu.md:88]: I'm not sure you need 'as proof of concept'.

File: content/cloud-servers/use-fail2ban-on-ubuntu.md (hunk 1 above)
Comment (catlook) [reply content/cloud-servers/use-fail2ban-on-ubuntu.md:29]: doh

File: content/cloud-servers/use-fail2ban-on-ubuntu.md (hunk 2 above)
Comment (catlook) [reply content/cloud-servers/use-fail2ban-on-ubuntu.md:88]: i debated.  good call

**Pull request merged**